website-warmup/task.json

```
@@ -70,18 +71,42 @@
             "helpMarkDown": "Ignore when the website has an error and continue"
         },
         {
-            "name": "BasicAuthUser",
+            "name": "Authentication",
+            "type": "pickList",
+            "label": "Select authentication type",
+            "required": false,
+            "defaultValue": "none",
+            "groupName": "auth",
+            "options": {
+                "basic": "basic",
+                "cred": "win cred",
```

Same here, and I would use the name windowscredentials as `crendentials` may be to generic
```suggestion
                "windowscredentials": "Windows Credentials",
```